src/app/components/SubHeading/index.jsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
+import { extractText } from '../../helpers/blocks';
 import { textPropTypes, textDefaultPropTypes } from '../../helpers/proptypes';
 
 const SubHeading = ({ blocks }) => {
-  const { text } = blocks[0].model.blocks[0].model;
+  const { text } = extractText(blocks);
+
+  if(!text){
```

Comment:
Surprised linting isn't complaining about this. A trivial change: add spacing to this conditional, e.g. `if (!text) {`